### PR TITLE
Bump rakudo-star to 2016.01

### PR DESCRIPTION
**Pull request by hoelzro**

### CODE REVIEW

File: library/rakudo-star [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # maintainer: Rob Hoelz <rob AT hoelz.ro> (@hoelzro)
 
-2015.11:  git://github.com/perl6/docker@1b836bce1461a6aac6cad32d3d33c12d9187f68f
+2016.01:  git://github.com/perl6/docker@ee6c27fe526dd73e2531ba3888ac3bfdcd0f4078
 
-latest:  git://github.com/perl6/docker@1b836bce1461a6aac6cad32d3d33c12d9187f68f
+2016.01:  git://github.com/perl6/docker@ee6c27fe526dd73e2531ba3888ac3bfdcd0f4078
```
Comment (yosifkit) [inline library/rakudo-star]: This should probably still be `latest:`. :wink:

**Comment by yosifkit** [comment]
Changeset: https://github.com/perl6/docker/compare/1b836bce...ee6c27fe52

File: library/rakudo-star (hunk 1 above)
Comment (hoelzro) [inline library/rakudo-star]: Derp, thanks for pointing that out!  Fixed!

**Comment by yosifkit** [comment]
LGTM, Build test of #1414; ac9a0ef (`rakudo-star`):

```console
$ bashbrew build "rakudo-star"
Cloning rakudo-star (git://github.com/perl6/docker) ...
Processing rakudo-star:2016.01 ...
Processing rakudo-star:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing rakudo-star:2016.01
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
```

**Pull request merged**